Assets/Scripts/Character/CharacterStateHandler.cs

```diff
@@ -49,6 +62,33 @@ public class CharacterStateHandler : NetworkBehaviour {
         currentState.PlayerCollisionEnter( collision );
     }
 
+    public GameObject GetPrefabInstanceFromType( PlayerType type ) {
+        string objectName;
+        switch (type) {
+            case PlayerType.PLAYER:
+                objectName = "PlayerNormal";
+                break;
+            case PlayerType.MONSTER:
+                objectName = "PlayerNormal";
+                break;
+            case PlayerType.AI_MONSTER:
+                objectName = "Toothy";
+                break;
+            case PlayerType.AI_PLAYER:
+                objectName = "AIPlayerNormal";
+                break;
+            default:
+                objectName = "AIPlayerNormal";
+                break;
+        }
+
+        string localPath = "Assets/Resources/Prefabs/Characters/" + objectName + ".prefab";
+        Object localInstance = AssetDatabase.LoadAssetAtPath( localPath, typeof( GameObject ) );
+        GameObject instance = PrefabUtility.InstantiatePrefab( localInstance ) as GameObject;
```

Comment:
Yeah trying to avoid that, this way also seems a lot easier to add more characters later. Like the player monster or more monster types